src/main/resources/python/api_client.hbs

```diff
@@ -1404,7 +1406,7 @@ class RequestBody(StyleFormSerializer, JSONDetector):
             if isinstance(cast_in_data, (schemas.FileIO, bytes)):
                 raise ValueError(f"Invalid input data type. Data must be int/float/str/bool/None/tuple/immutabledict and it was type {type(cast_in_data)}")
             return cls.__serialize_json(cast_in_data)
-        elif content_type == 'text/plain':
+        elif content_type in ['text/plain', 'application/x-pem-file']:
```

Comment:
How about using a set here and making it a class constant at the top of the class?
`__plain_txt_content_types = {'text/plain', 'application/x-pem-file'}`
`elif content_type in cls.__plain_txt_content_types:`

Reply:
Good point, I changed it.